src/spock_relcache.c

```
@@ -95,74 +96,48 @@ spock_relation_open(uint32 remoteid, LOCKMODE lockmode)
 
 				att = TupleDescAttr(desc, entry->attmap[i]);
-				dfunc = spock_lookup_delta_function(seclabel, att->atttypid);
+				fname = pstrdup(GET_STRING_RELOPTION(aopt,
+													 delta_apply_function));
+				dfunc = spock_lookup_delta_function(fname, att->atttypid);
+				pfree(fname);
 
 				if (dfunc == InvalidOid)
 					elog(ERROR, "SPOCK: column %s.%s.%s is configured for "
-						 "delta_apply function %s - function not found",
+						 "delta_apply_function %s - function not found",
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If pg_attribute_printf is a macro then please configure it.

(unknownMacro)